src/Logger/LoggerBase.php

```diff
@@ -85,25 +67,17 @@ public function catchMessages($log_entry) {
   }
 
   /**
-   * End message capturing.
-   *
-   * Effectively disables the shutdown function for the given log entry.
-   *
-   * @param \Drupal\ultimate_cron\Logger\LogEntry $log_entry
-   *   The log entry.
+   * {@inheritdoc}
    */
-  public function unCatchMessages($log_entry) {
+  public function unCatchMessages(LogEntry $log_entry) {
     $class = get_class($this);
     unset($class::$log_entries[$log_entry->lid]);
   }
 
   /**
-   * Invoke loggers watchdog hooks.
-   *
-   * @param array $log_entry
-   *   Watchdog log entry array.
+   * {@inheritdoc}
    */
-  final static public function hook_watchdog(array $log_entry) {
+  final static public function hook_watchdog(LogEntry $log_entry) {
```

Comment:
doesn't really make sense to have a final static function on an interface. also doesn't really make sense to have such a method here.. not sure what to do with it yet, let's add a note to our growing list.